api/resolver/query/project.go

```diff
@@ -0,0 +1,25 @@
+package query
+
+import (
+	"context"
+	"github.com/robojones/graphql/lib/session_context"
+	"github.com/robojones/graphql/prisma"
+)
+
+func (resolver *QueryResolver) Projects(ctx context.Context) ([]prisma.Project, error) {
+	user, err := session_context.User(ctx)
+
+	if err != nil {
+		return make([]prisma.Project, 0), err
```

Comment:
`return nil, err` is sufficient. `if err != nil`, the result is null anyway.

Reply:
Will fix that. I assumed slices were not nullable.